enterprise_access/apps/bffs/api.py

```diff
@@ -142,6 +144,13 @@ def get_and_cache_default_enterprise_enrollment_intentions_learner_status(
     response_payload = client.get_default_enterprise_enrollment_intentions_learner_status(
         enterprise_customer_uuid=enterprise_customer_uuid,
     )
+
+    # Don't set in the cache if there are enrollable intentions
+    if statuses := response_payload.get('enrollment_statuses', {}):
+        needs_enrollment = statuses.get('needs_enrollment', {})
+        if needs_enrollment.get('enrollment', []):
```

Comment:
this shouldn't say "enrollable", should it?
```suggestion
        if needs_enrollment.get('enrollable', []):
```

Reply:
ah, yes, good call